internal/admin/dashboard/templates/sidebar.html

```
@@ -40,6 +40,10 @@ <h1>GoModel</h1>
             <i data-lucide="workflow" class="nav-icon" aria-hidden="true"></i>
             <span>Workflows</span>
         </a>
+        <a href="{{appURL "/admin/dashboard/failover"}}" class="nav-item" :class="{ active: page === 'failover' }" title="Failover" @click.prevent="navigate('failover')">
```

HTMLHint (1.9.2)

[error] Special characters must be escaped : [ < ].

(spec-char-escape)